Use `arb-dir`

kzrnm opened the pull request:
Fixes #50

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

mosuem reviewed (changes requested):
Could you add a test for this?

Thanks for the PR!

kzrnm commented:
@mosuem 
I updated behavior and added testing.

## Behavior

https://github.com/google/arb-editor/blob/928c50eb396c5ef7e1f0994ae7681afde56c5c34/src/messageParser.ts#L201-L202
 
Default values for `arb-dir` and `template-arb-file` are now `lib/l10n` and `app_en.arb`, respectively, if they do not exist. This is a breaking change for arb-editor, but it satisfies the official arb behavior.

https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization#configuring-the-l10n-yaml-file

## Testing

The original `parseAndDecorate` function was not suitable for testing, so I changed it to a method of the `Parser` class. This conflicts with #45.

mosuem commented:
The workflows seem to be failing on main as well - I am investigating.
